app/services/config-persistence/config.ts

```diff
@@ -62,8 +62,13 @@ export class ConfigPersistenceService extends PersistentStatefulService<IScenesC
     if (!fs.existsSync(this.configFileDirectory)) return;
 
     const configsNames = fs.readdirSync(this.configFileDirectory).map(file => file.replace(/\.[^/.]+$/, ''));
+    const configsNamesWithoutBackup: string[] = [];
+
     if (configsNames.length) {
-      this.ADD_SCENES_COLLECTIONS(configsNames);
+      configsNames.forEach (configName => {
+        if (configName.indexOf('_backup') === -1) configsNamesWithoutBackup.push(configName);
```

Comment:
I think we should do what OBS does and use a `.json.bak` file extension for backup files.  The way you have this written, any scene collection with `_backup` in the name will be treated as a backup, even if that's just what the user named it.

Reply:
Done.